FirebaseRemoteConfig/Tests/Unit/RCNRemoteConfigTest.m

```diff
@@ -218,58 +218,60 @@ - (void)tearDown {
 }
 
 - (void)testFetchConfigWithNilCallback {
+  __block NSMutableArray<FIRRemoteConfig *> *configInstances = _configInstances;
```

Comment:
How does this help? `_configInstances` is a member and the test should be waiting for all expectations before exiting the method body. This should already ensure that everything lives as long as it should.

Reply:
What happens when other tests start up in parallel and their setup call reassigns `_configInstances`?

Reply:
OK, still failing, so it's something else - https://travis-ci.org/firebase/firebase-ios-sdk/jobs/576247815

Reply:
Each test instance gets its own instance of the test fixture. They don't interfere with each other.

Reply:
Thanks. I'll close this one.